BrandonAinsworth/FitLit
# Refactor codebase and remove unused lines and files

#### What's this PR do?

Removes unnecessary code, console logs and whitespace

#### Where should the reviewer start?

Wherever the reviewer pleases. (Every file changed here)

#### How should this be manually tested?

NPM test from the root directory, also run NPM start to check it on the browser at localhost:8080;

#### Any background context you want to provide?

Woohoo!
